bool plan_status
string plan_name
